pkg/service/healthcheck/metrics.go

```diff
@@ -8,57 +8,85 @@ import (
 )
 
 const (
-	clusterKey  = "cluster"
-	hostKey     = "host"
-	dcKey       = "dc"
-	pingTypeKey = "ping_type"
-	rackKey     = "rack"
+	clusterKey = "cluster"
+	dcKey      = "dc"
+	rackKey    = "rack"
+	hostKey    = "host"
```

Comment:
just out of curiosity: Why ping_type label has been removed?

Reply:
It wasn't used anywhere in the code, so I deleted it.

I think that in the past we had a single metric for all types of healthchecks (REST/CQL/alternator) with a label defining its type. Now we have different metrics for different healthcheck types, but someone probably forgot to remove this ping_type key .

Reply:
got it, thanks!